Composer/packages/client/src/recoilModel/selectors/formDialogs.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I would name the file dialogImports (it isn't completely form dialogs related)

Reply:
It can be soon, so I'd keep it this way.

Reply:
My point is that the LU/LG imports won't be behind the form dialog feature switch - they are a general improvement to support users editing imports.